import-scripts/importUsers.py

```diff
@@ -627,10 +630,14 @@ def main():
                                                 portal_properties.google_worksheet)
 
             # the 'guts' of the script
-            new_user_map, emails_to_remove = manage_users(client, google_spreadsheet, cursor, worksheet_feed, portal_name_map[google_spreadsheet], mskcc_user_spreadsheet)
+	    # special case genie-archive because
+	    # original script depended on one to one mapping of spreadsheet to app name - and lookup was by spreadsheet
+	    # with genie-archive we want to do one to many mapping (one spreadsheet to multiple apps)
+	    # to fit this logic would have to rework how we specify properties or introduce new column (db name) as index but might have other effects
+            new_user_map, emails_to_remove = manage_users(client, google_spreadsheet, cursor, worksheet_feed, GENIE_ARCHIVE_APP_NAME if portal_properties.cgds_database_name == GENIE_ARCHIVE_DB_NAME else portal_name_map[google_spreadsheet], mskcc_user_spreadsheet)
```

Comment:
maybe do the logic for using GENIE_ARCHIVE_APP_NAME once and store the result in a variable (app_name?) Then you can use the variable in both locations